token.go

```diff
@@ -19,6 +19,10 @@ import (
 // expirations due to client-server time mismatches.
 const expiryDelta = 10 * time.Second
 
+// These are the fields which we normally expect to be in a token,
+// so we don't include them with requests for the Extra fields.
+var OAUTH_FIELDS = []string{"access_token", "token_type", "refresh_token", "expires_in", "expires"}
```

Comment:
Minor nit, but:

- Various linters will complain about all caps and underscores in the name, so it should probably be changed
- The rest of this package looks like it follows godoc convention, in which case this being exported it should be documented properly

With that said, it likely does *not* need to be exported and could be changed to something like `standardTokenFields`, which would both make it private and satisfy any doc linters.

Reply:
Who is this Various Linters guy and why is he all in my business?

Fixed, thanks @avidal.